### ISSUE TITLE
Play plugin possibly broken after upgrade to Play 2.3.8

### ISSUE THREAD

**Issue by Eduardo-M-Cavalcanti** [scala-ide/scala-ide-play2#244]
Compiler fails with messages such:
```
object "template_name" is not a member of package views.html
```
when compilling controllers that refer to templates in the form views.html."template_name".

Play 2.3.6, 2.3.7 not affected.

**Comment by mkurz** [comment]
Maybe [this anwer from stackoverflow](http://stackoverflow.com/a/28550840/810109) helps?

**Comment by Eduardo-M-Cavalcanti** [comment]
Adding 
```
<classpathentry kind="src" path="target/scala-2.11/twirl/main"/>
```
to .classpath manually solved the issue.
Thank you.

**Comment by benmccann** [comment]
This is fixed in Play 2.4. It's a Play and sbteclipse issue. I think we can close this issue since it's not Scala IDE's fault.

See this issue instead: https://github.com/playframework/playframework/issues/4078

**Comment by dotta** [comment]
@benmccann Agreed.
